package.json

```diff
@@ -26,7 +26,8 @@
   },
   "devDependencies": {
     "jshint": "*",
-    "mocha": "*"
+    "mocha": "*",
+    "fs-extra": "*"
```

Comment:
Let's stick with the default features `fs` and `yeoman-generator` gives us. Or is there a reason not to?